pilot/docker/Dockerfile.proxyv2

```diff
@@ -4,6 +4,14 @@ ARG BASE_DISTRIBUTION=default
 # Version is the base image version from the TLD Makefile
 ARG BASE_VERSION=latest
 
+# This builder image has /sbin/setcap installed.
+FROM docker.io/istio/base:${BASE_VERSION} as libpcapbuilder
```

Comment:
should we just add this to the base image? On my machine its 15kb so seems pretty tiny?

Reply:
Sounds good to me.

Reply:
Sent https://github.com/istio/istio/pull/20804 out for review.

Reply:
Done. Updated to use base version 1.6-dev.1, which has `libcap2-bin` installed.